lib/compat/wordpress-6.3/class-gutenberg-rest-global-styles-controller-6-3.php

```diff
@@ -0,0 +1,184 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Global_Styles_Controller class
+ *
+ * @package    Gutenberg
+ * @subpackage REST_API
+ */
+
+/**
+ * Base Global Styles REST API Controller.
+ */
+class Gutenberg_REST_Global_Styles_Controller_6_3 extends Gutenberg_REST_Global_Styles_Controller_6_2 {
+	/**
+	 * Registers the controllers routes.
+	 *
+	 * @since 6.3 Registers `/revisions` endpoint.
+	 *
+	 * @return void
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/(?P<id>[\/\w-]+)/revisions',
```

Comment:
TODO: add schema

Reply:
Needs to follow this pattern. 
https://github.com/WordPress/wordpress-develop/blob/fea66becf567d3cdcaa40d6edbec6e0d20df362e/src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php#L89